fieldtracing/fieldtracing.cpp

```diff
@@ -674,8 +674,9 @@ namespace FieldTracing {
          fsgridCell = getLocalFsGridCellIndexForCoord(technicalGrid,{(Real)x[0], (Real)x[1], (Real)x[2]});
          
          // If we map into the ionosphere, discard this field line.
-         if(x.at(0)*x.at(0) + x.at(1)*x.at(1) + x.at(2)*x.at(2) < fieldTracingParameters.innerBoundaryRadius*fieldTracingParameters.innerBoundaryRadius) {
-            cellTracingCoordinates[n] = {0,0,0};
+         if(x.at(0)*x.at(0) + x.at(1)*x.at(1) + x.at(2)*x.at(2) < //fieldTracingParameters.innerBoundaryRadius*fieldTracingParameters.innerBoundaryRadius) {
+            SBC::Ionosphere::innerRadius*SBC::Ionosphere::innerRadius ) { 
+            cellTracingCoordinates[n] = x;
```

Comment:
This breaks support for conductingsphere cases.

Reply:
So please fiddle with where fieldTracingParameters.innerBoundaryRadius gets assigned. Now this would be also an argument in favour of continuing with analytic inside of the vg radius.

Reply:
So how's this now?